### ISSUE TITLE
BatchNormalization implementation for float64.

### ISSUE THREAD

**Issue by AlirezaAlizadeh** [microsoft/onnxruntime#2874]
**Is your feature request related to a problem? Please describe.**

I try to load an onnx exported torch model with `BatchNormalization` layer in an onnxruntime session. This works when I use `float32` tensors. However, I get the error (`[ONNXRuntimeError] : 9 : NOT_IMPLEMENTED : Could not find an implementation for the node BatchNormalization(9)`) in case I use `float64` tensors.

**System information**
- ONNX Runtime version: 1.1.0
- Torch version: 1.4.0

**Describe the solution you'd like**
It should be possible to load onnx models with `BatchNormaliziation` layers of type `float64`.

**Describe alternatives you've considered**
Tensors with `float32` type could be used instead.

**Additional context**
This code snippet reproduces the mentioned issue: 

``` 
import torch
import onnxruntime as rt

class test_module(torch.nn.Module):
    def __init__(self):
        super(test_module, self).__init__()
        self.bn = torch.nn.BatchNorm1d(128, affine=False).double()
    def forward(self, x):
        B,N,D = x.shape
        x = x.view(-1, D)
        x = self.bn(x.double())
        x = x.view(B,N,D)
        return x
dummy_x= torch.randn(4,16,128)
net = test_module()
torch.onnx.export(net, (dummy_x),
                "tmp.onnx",
                input_names=['input'],
                output_names=['preds'],
                verbose=True,
         )

rt.InferenceSession("tmp.onnx")
```

**Comment by pranavsharma** [comment]
Would you like to contribute? Should be an easy enhancement.

**Comment by prabhat00155** [comment]
I can look into this.

**Comment by AlirezaAlizadeh** [comment]
Thanks for considering that. I can spend time on that in the next week. However, I'm also fine if you wanna look into it @prabhat00155.

**Comment by prabhat00155** [comment]
@pranavsharma Do you know how do I build a python whl file for onnxruntime?

**Comment by snnn** [comment]
Fixed in #2941
